src/components/NavBar/index.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { NotificationIcon, AvatarIcon, SearchIcon } from '../../assets/icons';
+import './NavBar.scss';
+import logo from '../../assets/images/logo.png';
+
+const NavBar = ({ isLoggedIn, user }) => {
+  const [menuOpen, setMenuOpen] = React.useState(false);
+
+  const toggleMenu = () => {
+    setMenuOpen(!menuOpen);
+  };
+  return (
+    <nav className="wrapper flex flex-row items-center justify-between pt-2 pb-2 md:pl-10 md:pr-10 sm:pl-4 sm:pr-4">
+      <div className="logo flex items-center">
+        <Link to="/" title="Error Swag Logo" className="flex items-center">
+          <img
+            src={logo}
+            alt="Errorswag logo"
+            className="sm:w-8/12 md:w-full"
+          />
+        </Link>
+      </div>
+      <div className="flex">
+        <Link to="/search" className="small-button" title="Search">
```

Comment:
You can use any of these.
aria-labelledby
aria-label
aria-describedby (for long description. announced afterwards)
title

Title is actually the least supported. The others are preferred over title by screen readers